### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by sonuengineer**
- Add code for dynamic portfolio and pop window for work-section
- update the style.css for a popup window
- update the index.html

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -599,6 +595,12 @@ div.about {
     background-image: url("../imgs/about-mebg-image_desktop.png");
     background-repeat: no-repeat;
     background-position-x: right;
+
+    /* position: absolute;
+    height: 354px;
+    width: 586px;
+    left: 37%;
+    top: 5%; */
```
Comment (AmaduKamara) [inline css/styles.css]: - [x] Kindly remove all unused CSS codes from your codebase. It's good for your project to have a clean, professional, and lean codebase. Having commented on code makes it dirty. Please remove them 👍

File: script.js [hunk 2]
```diff
@@ -30,3 +30,211 @@ contactLink.addEventListener('click', () => {
...
+    liveLink: 'https://sonuengineer.github.io/MyPortfolio/',
+    sourceLink: 'https://github.com/sonuengineer/MyPortfolio',
+  },
+  {
+    id: 'work1',
+    name: 'Profesional Art Printing Data',
+    description: `A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard.`,
+    featImage: '../imgs/placeholder-img.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
```
Comment (AmaduKamara) [inline script.js]: - [x] Currently, the path for the featImage property here is not working. Having a `../imgs` means you are moving out of a directory and into another directory. But technically, you want to go into a directory here, from the `script.js`, and hence, it should be `./imgs`. Remove one dot from this path here so the images load and display on the popup modal properly 👍

File: script.js [hunk 3]
```diff
@@ -30,3 +30,211 @@ contactLink.addEventListener('click', () => {
...
+    liveLink: 'https://sonuengineer.github.io/MyPortfolio/',
+    sourceLink: 'https://github.com/sonuengineer/MyPortfolio',
+  },
+  {
+    id: 'work2',
+    name: 'Data Dashboard Healthcare',
+    description: `A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard.`,
+    featImage: '../imgs/placeholder-img.png',
+    technologies: ['html', 'javascript', 'css'],
```
Comment (AmaduKamara) [inline script.js]: - [x] Currently, the path for the featImage property here is not working. Having a `../imgs` means you are moving out of a directory and into another directory. But technically, you want to go into a directory here, from the `script.js`, and hence, it should be `./imgs`. Remove one dot from this path here so the images load and display on the popup modal properly 👍

File: script.js [hunk 4]
```diff
@@ -30,3 +30,211 @@ contactLink.addEventListener('click', () => {
...
+    liveLink: 'https://sonuengineer.github.io/MyPortfolio/',
+    sourceLink: 'https://github.com/sonuengineer/MyPortfolio',
+  },
+  {
+    id: 'work3',
+    name: 'Website Protfolio',
+    description: `A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard.`,
+    featImage: '../imgs/placeholder-img.png',
+    technologies: ['html', 'javascript', 'Ruby'],
```
Comment (AmaduKamara) [inline script.js]: - [x] Currently, the path for the featImage property here is not working. Having a `../imgs` means you are moving out of a directory and into another directory. But technically, you want to go into a directory here, from the `script.js`, and hence, it should be `./imgs`. Remove one dot from this path here so the images load and display on the popup modal properly 👍

File: script.js [hunk 5]
```diff
@@ -30,3 +30,211 @@ contactLink.addEventListener('click', () => {
...
+    liveLink: 'https://sonuengineer.github.io/MyPortfolio/',
+    sourceLink: 'https://github.com/sonuengineer/MyPortfolio',
+  },
+  {
+    id: 'work4',
+    name: 'Profesional Art Data Printing',
+    description: `A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard.`,
+    featImage: '../imgs/placeholder-img.png',
+    technologies: ['html', 'css', 'Ruby'],
```
Comment (AmaduKamara) [inline script.js]: - [x] Currently, the path for the featImage property here is not working. Having a `../imgs` means you are moving out of a directory and into another directory. But technically, you want to go into a directory here, from the `script.js`, and hence, it should be `./imgs`. Remove one dot from this path here so the images load and display on the popup modal properly 👍

File: script.js [hunk 6]
```diff
@@ -30,3 +30,211 @@ contactLink.addEventListener('click', () => {
...
+    liveLink: 'https://sonuengineer.github.io/MyPortfolio/',
+    sourceLink: 'https://github.com/sonuengineer/MyPortfolio',
+  },
+  {
+    id: 'work5',
+    name: 'Healthcare Data Dashboard',
+    description: `A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard.`,
+    featImage: '../imgs/placeholder-img.png',
+    technologies: ['html', 'javascript', 'bootstrap'],
```
Comment (AmaduKamara) [inline script.js]: - [x] Currently, the path for the featImage property here is not working. Having a `../imgs` means you are moving out of a directory and into another directory. But technically, you want to go into a directory here, from the `script.js`, and hence, it should be `./imgs`. Remove one dot from this path here so the images load and display on the popup modal properly 👍

File: script.js [hunk 7]
```diff
@@ -30,3 +30,211 @@ contactLink.addEventListener('click', () => {
...
+    liveLink: 'https://sonuengineer.github.io/MyPortfolio/',
+    sourceLink: 'https://github.com/sonuengineer/MyPortfolio',
+  },
+  {
+    id: 'work6',
+    name: 'My Website Portfolio',
+    description: `A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard.`,
+    featImage: '../imgs/placeholder-img.png',
+    technologies: ['html', 'css', 'bootstrap', 'Ruby'],
```
Comment (AmaduKamara) [inline script.js]: - [x] Currently, the path for the featImage property here is not working. Having a `../imgs` means you are moving out of a directory and into another directory. But technically, you want to go into a directory here, from the `script.js`, and hence, it should be `./imgs`. Remove one dot from this path here so the images load and display on the popup modal properly 👍

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @sonuengineer,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly 👍 
✅ Linter checks passed 👍 
✅ Loading popup data dynamically 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by mukhammadrizooff — APPROVED** [review]
Hi @sonuengineer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**